sdk/storage/Azure.Storage.Blobs/src/Models/ContentRange.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Text;
+
+namespace Azure.Storage.Blobs.Models
+{
+    internal struct ContentRange
+    {
+        private const string WildcardMarker = "*";
+
+        public struct RangeUnit
+        {
+            internal const string BytesValue = "bytes";
+
+            private readonly string _value;
+
+            /// <summary>
+            /// Initializes a new instance of the <see cref="RangeUnit"/> structure.
+            /// </summary>
+            /// <param name="value">The string value of the instance.</param>
+            public RangeUnit(string value)
+            {
+                _value = value ?? throw new ArgumentNullException(nameof(value));
+            }
+
+            /// <summary>
+            /// AES-CBC using a 256 bit key.
```

Comment:
this class doesn't seem to be dependent on algorithm. this information may find better home.

Reply:
Copy/paste error. Property is necessary but comment is irrelevant.